doc/reference/reference_lua/compat/box_info_cluster_meaning.rst

```diff
@@ -0,0 +1,72 @@
+.. _compat-option-box-info-cluster:
+
+Meaning of box.info.cluster
+===========================
+
+Option: ``box_info_cluster_meaning``
+
+Starting from version 3.0, the :ref:`box_info_cluster` table stores the information
+about the entire cluster. In earlier versions, it stored only the current replica set
+information. The ``box_info_cluster_meaning`` compat option allows to define the
+the scope of ``box.info.cluster``: the entire cluster or a single replica set.
```

Comment:
Can we add something that makes it clear that the new behavior is 3.0.0's default?

After reading this paragraph I was in a doubt: whether it is 3.0.0's behavior that can be enabled using the compat option or it is a default in this version.